services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -288,7 +286,9 @@ def get_healthcheck_timeout_in_seconds(cls, v):
     # HELPERS  --------------------------------------------------------
 
     def is_enabled(self, field_name: str) -> bool:
-        return bool(getattr(self, field_name, None))
+        return bool(
+            getattr(self, field_name, None)
+        )  # should we also check `self.field_name.enabled == False`?
```

Comment:
Is this a question to us? a note already there?
```
# should we also check `self.field_name.enabled == False`?
```
Answer: No, we dropped `.enable` in the first version. You might see something from the old configs still in the fixtures of the tests. 
The new policy is: a field of the ApplicationSettings, representing plugin settings can be disabled if:
a) is nullable or bool
b) evaluates False